app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+  def index
+    @latest_books = Book.includes(:authors).latest(3).decorate
+    @best_sellers = BestSellers.new.query.map(&:decorate)
```

Comment:
don't forget after making result a relation to use `#decorate` on result

Reply:
fixed